.ci/.bump-go-release-version.yml

```diff
@@ -50,5 +50,7 @@ projects:
     script: .ci/bump-go-release-version.sh
     branches:
       - main
+      - "1.17"
```

Comment:
It's not yet available, but let's add this here to help the automation in the future